.github/ISSUE_TEMPLATE/question.md

```diff
@@ -6,4 +6,4 @@ labels: question
 
 # Question
 
-Use [Github Discussions](https://github.com/open-telemetry/opentelemetry-dotnet/discussions/new).
+Use [Github Discussions](https://github.com/open-telemetry/opentelemetry-demo-webstore/discussions/).
```

Comment:
@open-telemetry/demo-webstore-maintainers please advise if we should create/use Discussions or something else.

Reply:
I think we should use / create it

Reply:
Okay, I'll merge this PR to unblock the CI. Then you can help to make these CI `required`.

We can discussion more in the upcoming SIG meeting if folks have other ideas.